pages/workflow/PKPWorkflowHandler.php

```diff
@@ -439,10 +439,10 @@ public function index($args, $request)
             'identifiersEnabled' => $identifiersEnabled,
             'metadataEnabled' => $metadataEnabled,
             'pageComponent' => 'WorkflowPage',
-            'pageTitle' => join(__('common.titleSeparator'), [
+            'pageTitle' => implode(__('common.titleSeparator'), array_filter([
                 $submission->getShortAuthorString(),
                 $submission->getLocalizedTitle()
-            ]),
+            ])),
```

Comment:
What's the reasoning for the change here and in `PKPAuthorDashboardHandler`?

Reply:
I needed a way to create the title of the page by using the strings provided, but with not having to check whether the `$submission->getShortAuthorString()` is empty or not. 

That way if the `$submission->getShortAuthorString()` is empty then the `implode` returns `$submission->getLocalizedTitle()` with no separator. If the `$submission->getShortAuthorString()` is not empty, the `implode` returns `author | title` separated with `|`

Reply:
Ah, right! Clever. :+1: